liberty-support-feature/README.md

```diff
@@ -1,6 +1,6 @@
 # Arquillian support Liberty user feature
 
-A liberty user feature which allows deployment exceptions to be reported more reliably when using the liberty-managed container
+A liberty user feature which allows deployment exceptions to be reported more reliably when using the liberty-managed container.
```

Comment:
Although not part of the current changes, capitalize Liberty and Arquillian when they are used as standalone words in this document.